spec/system/admin/products_spec.rb

```diff
@@ -64,6 +64,42 @@
       expect(page).to have_content "Name can't be blank"
     end
 
+    it "display all attributes when submitting with error: Unit Value must be grater than 0" do
+      login_to_admin_section
+
+      click_link 'Products'
+      click_link 'New Product'
+
+      select @supplier.name, from: 'product_supplier_id'
+      fill_in 'product_name', with: "new product name"
```

Comment:
It's better to use the labels (if they are connected to the inputs correctly).

Reply:
Some labels are not connected correctly.
For example "display as"
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/16113997/7eb25c40-2d1e-4c86-af75-8301dab8af16)

Reply:
That's a shame. We should fix that. Out of scope here though.